CASC/Schedules.cpp

```diff
@@ -16,10 +16,29 @@
 
 #include "Schedules.hpp"
 
+#include <fstream>
+
 using namespace Lib;
 using namespace Shell;
 using namespace CASC;
 
+void Schedules::getScheduleFromFile(const vstring& filename, Schedule& quick)
+{
+  if (filename == "") {
+    USER_ERROR("Schedule file was not set.");
```

Comment:
I wonder if the default / invalid value can ever be "<empty>".

Reply:
I think this is fine. If somebody supplies an empty string somehow they probably don't have a file with an empty name.